### DISCUSSION TITLE
登录节点失败

### DISCUSSION THREAD

**Discussion by linyiyi1314** [natfrp/wiki#451] [使用问题]
使用校园网创建mc节点，但一直显示登录节点失败，切换流量才可以使用，网络是网线 使用WIFI也不行 必须要使用流量才可以，管理员运行此软件也行不通
<img width="593" alt="微信图片_20240327212237" src="https://github.com/natfrp/wiki/assets/138667156/49dda0c1-faad-4bab-bcff-ee83f6c5adb7">

**Comment by fakeboboliu** [comment]
此问题通常因为校园网管理人员为了**维护网络安全**屏蔽了我们服务。

如果您确定
- **_您的行为不涉及任何有风险的数据_**
- **_您穿透的服务不存在被控制泄密的风险_**
- **_且您愿意承担出现泄密的任何后果_**

您可以考虑在设置中打开使能 TLS 开关，这个开关可能会绕过此屏蔽。

**Reply by linyiyi1314** [reply]
好的 解决了
